docs/UserGuide.md

```diff
@@ -7,12 +7,21 @@
   - [Planner Mode](#planner-mode)
```

Comment:
I'm not sure if we should leave this as planner mode.. I think we should standardise what we call it since it's called timetable mode in our code.